hail/python/hail/matrixtable.py

```diff
@@ -1941,7 +1941,7 @@ def unfilter_entries(self):
         entry_ir = hl.if_else(
             hl.is_defined(self.entry),
             self.entry,
-            hl.literal(hl.Struct(**{k: hl.missing(v.dtype) for k, v in self.entry.items()})))._ir
+            hl.struct(**{k: hl.missing(v.dtype) for k, v in self.entry.items()}))._ir
```

Comment:
this avoids an `eval` just to get `None` for the missing (which is the converted back to a missing IR)